bench/is_types.py

```diff
@@ -51,7 +53,7 @@ def cache_type_e_is_Dict(e):
 
 def cache_type_e_is_List(e):
     t = type(e)
-    return t is list or isinstance(e, UserList)
+    return t is list or isinstance(e, UserList or isinstance(e, deque))
```

Comment:
Is this right? `or` inside the isinstance()?

Reply:
that got clumsy. I'll fix - think should be single isinstance

Reply:
yeah, it was supposed to look like the one a few lines later.  pushed update.